doc/docs_es.md

```diff
@@ -0,0 +1,116 @@
+# Documentación de V
+
+## Introducción
+
+V es un lenguaje de programación compilado de tipado estático diseñado para crear software mantenible.
+
+Es similar a Go y su diseño también ha sido influenciado por Oberon, Rust, Swift,
+Kotlin y Python.
+
+V es un lenguaje muy simple. Leer esta documentación te llevará aproximadamente una hora,
+y al final habrás aprendido prácticamente todo el idioma.
```

Comment:
I've never heard a programming language called an `idioma de programación`, always `lenguaje de programación`. To me `idioma` == spoken language (maybe that's a regional thing tho, not sure)

Reply:
No, it's another mistake of mine.

Reply:
Spanish is one of the most diverse languages in terms of dialects :)

Reply:
It is fixed. Thanks for the help :).

Reply:
Are you spanish @spaceface777 ?

Reply:
I am.